packages/domain/lauréat/src/garantiesFinancières/mainLevée/annuler/annulerMainLevéeGarantiesFinancières.behavior.ts

```diff
@@ -0,0 +1,87 @@
+import { DateTime, Email, IdentifiantProjet } from '@potentiel-domain/common';
+import { DomainEvent, InvalidOperationError } from '@potentiel-domain/core';
+
+import { GarantiesFinancièresAggregate } from '../../garantiesFinancières.aggregate';
+
+export type MainLevéeGarantiesFinancièresAnnuléeEvent = DomainEvent<
+  'MainLevéeGarantiesFinancièresAnnulée-V1',
+  {
+    identifiantProjet: IdentifiantProjet.RawType;
+    annulation: {
+      annuléLe: DateTime.RawType;
+      annuléPar: Email.RawType;
+    };
+  }
```

Comment:
```suggestion
  {
    identifiantProjet: IdentifiantProjet.RawType;
    annuléLe: DateTime.RawType;
    annuléPar: Email.RawType;
  }
```

c'est pas nécessaire d'avoir la clé "annulation" et aussi pour rester cohérents avec les autres events (ex : AbandonAnnulé)